start.py

```diff
@@ -290,7 +290,11 @@ def main(args):
     logger.info('Performing health check on Pulsar cluster (%s) ...', args.pulsar_cluster_name)
     def condition(node, cluster_name, command):
         command_status = node.execute(command, quiet=True)
-        return command_status.exit_code == 0 and command_status.output.strip() == cluster_name
+        # At some point Pulsar changed output format to enclose cluster name with quotes, so we are removing them if they are there
+        listed_cluster_name = command_status.output.strip()
```

Comment:
`str.strip`, if no args are given, will strip for whitespace. If args (a string) is given, those characters will be stripped from the start and end of the string. Perhaps we might wanna do `command_status.output.strip(' "')` to remove any spaces and the double quotes in one command?

Reply:
Thank you for the quick python lesson :)